cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -539,13 +549,52 @@ static int partitionBoundary(HttpServletRequest request, boolean isUpperBound) {
   /**
    * Default: An empty set.
    */
-  static Set<UUID> userTaskIds(HttpServletRequest request) throws UnsupportedEncodingException {
+  public static Set<UUID> taskIds(HttpServletRequest request) throws UnsupportedEncodingException {
```

Comment:
Here we are parsing a parameter called `user_task_ids` (specified via `USER_TASK_IDS_PARAM`), but this function seems to retrieve `taskIds`. Due to our convention, we would like ensure that the parameter names, their getters/parsers, and the user of these functions use the same name -- which, in this case, is `userTaskIds` (not `taskIds`).